cli/src/commands/file.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020-2023 The Jujutsu Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::cli_util::CommandHelper;
+use crate::command_error::CommandError;
+use crate::commands::{file_chmod, file_print};
+use crate::ui::Ui;
+
+/// File operations.
+#[derive(clap::Subcommand, Clone, Debug)]
+pub enum FileCommand {
+    #[command(visible_alias("cat"))]
```

Comment:
We probably don't need this alias? At least we don't need it for backwards compatibility.

Reply:
it isn't needed for back compat, but I do think it is common enough in the linux world that we should keep it for convenience

Reply:
For Unix folks, I think the top-level `jj cat` alias is more important. I personally don't think Unixy command name in namespaced command would improve discoverability.

Reply:
removed for now, we can add it back later